report/sections/system-design.tex

```diff
@@ -34,10 +35,205 @@ \subsection{Model}
+\tikzset{
+    block/.style = {draw, rectangle, 
+    minimum height=1cm, 
+    minimum width=2cm},
+    input/.style = {coordinate,node distance=1cm},
+    output/.style = {coordinate,node distance=2cm},
+    arrow/.style={draw, -latex,node distance=2cm},
+    pinstyle/.style = {pin edge={latex-, black,node distance=2cm}},
+    sum/.style = {draw, circle, node distance=1cm}
+}
+\begin{figure}[h]
+    \begin{center}
+        \begin{tikzpicture}[auto, node distance=2.5cm,>=latex']
+        \node [input, name=input] {};
+        \node [sum, right of=input] (sum) {};
+        \node [block, right of=sum] (controller) {$PID$};
+        \node [block, right of=controller] (plant) {$G(s)$};
+        \node [output, right of=plant] (output) {};
+        \node [block, below of=plant] (feedback) {$\frac{1}{s}$};
+        \draw [draw,->] (input) -- node {$U(s)$} (sum);
+        \draw [->] (sum) -- node {} (controller);
+        \draw [->] (controller) -- node {} (plant);
+        \draw [->] (plant) -- node [name=y] {$Y(s)$}(output);
+        \draw [->] (y) |- node [above,pos=0.79] {} (feedback) ;
+        \draw [->] (feedback) -| node[pos=0.99] {$-$} 
+        node [near end] {} (sum);
+        \end{tikzpicture}
+    \end{center}
+    \caption{Closed Loop system with a integrator in the feedback.}\label{fig:closedloop}
+\end{figure}
+
+In this case the closed loop transfer function will be as shown in Eq.(\ref{eq:closedloop})
+\begin{equation}
+G_{c}(s) = \frac{PID}{s + PID}
+\label{eq:closedloop}
+\end{equation} 
+
+As the closed loop transfer function of the system is describing a first order system, it is possible to use a P-controller. 
+I.e. the controller only has a proportional gain and doesn't require a integrative or derivative elements. 
+So the final closed loop transfer function will be 
+\begin{equation}
+G_{c}(s) = \frac{K_p}{s + K_p}
+\label{eq:kp}
+\end{equation}
+
+This system will have a pole at $-K_p$.
+Any positive proportional gain will results in a stable pole and which in turn a stable system. \\
+\begin{figure*}[ht]
+\centering
+\includegraphics[width=\textwidth]{sections/assets/Theta_PID.PNG}
+\caption{The complete control block for the angular position in discrete time domain.}
+\label{fig:PID1}
+\end{figure*}
+\\
+ 
+It is required to make sure that $\theta$ error, $\theta_e$ is in $(-\pi,\pi]$ span. Due to the $\theta = \theta \pm 2 \cdot n \cdot \pi$, it is possible for the error be in out of this span.
+By using a function called arctan2 or atan2 in many programming languages it is possible to accomplish this challenge.\\
+This function often has two input argument (y,x).\\ 
+By defining 
+\begin{equation}
+\theta = atan2(sin(\theta),\, cos(\theta))
+\end{equation}
+$\theta$ will always be in $(-\pi,\pi]$ span.\\
+
+Due to the physical limits the servos have, two saturation blocks are required.
+These blocks are shown in figure(\ref{fig:PID1}) and the assumption for their limits are (-10,10) $[cm/sec]$. 
```

Comment:
Units are not to be italicized. Use SI units. `(-10,10) $[cm/sec]$` -> `$(-10,10)$~[cm/s]`.